charmcraft/templates/init-simple/tests/unit/test_charm.py.j2

```diff
@@ -4,42 +4,25 @@
 # Learn more about testing at: https://juju.is/docs/sdk/testing
 
 import unittest
-from unittest.mock import Mock
 
-from ops.model import ActiveStatus
+import ops.testing
+from ops.model import ActiveStatus, BlockedStatus, WaitingStatus
 from ops.testing import Harness
 
 from charm import {{ class_name }}
 
+# Enable more accurate simulation of container networking.
+# For more information, see https://juju.is/docs/sdk/testing#heading--simulate-can-connect
+ops.testing.SIMULATE_CAN_CONNECT = True
```

Comment:
this should probably be a pytest fixture autoused at the module scope, in the future someone might find themselves scratching their heads if they do something like `tests.integration.test_...` and this setting might get in the way

Reply:
good idea!

Reply:
actually I changed to make it similar to current test format (put it in the `setUp` of the class)